monkey/monkey_island/cc/next_ui/src/redux/rootReducer.ts

```diff
@@ -1,11 +1,11 @@
 import { combineReducers } from 'redux';
-import { externalApiSlice } from '@/redux/features/api/externalApiSlice';
-import { internalApiSlice } from '@/redux/features/api/internalApiSlice';
+import { islandApiSlice } from '@/redux/features/api/islandApiSlice';
 import themeSliceReducer from './features/theme/theme.slice';
+import authenticationTimerSlice from '@/redux/features/api/authentication/authenticationTimerSlice';
 
 const rootReducer = combineReducers({
-    [externalApiSlice.reducerPath]: externalApiSlice.reducer,
-    [internalApiSlice.reducerPath]: internalApiSlice.reducer,
+    [islandApiSlice.reducerPath]: islandApiSlice.reducer,
```

Comment:
I think that `theme.slice.ts` should be renamed, but since it has nothing to do with authentication I'm postponing this.
API slice and just a slice have a bit different interface. 

I think this is to be expected, because API slice and just slice have different interfaces: https://redux.js.org/tutorials/essentials/part-7-rtk-query-basics#configuring-the-store